### PR TITLE
[FR] Add "Featured Artist playlists"

### PR DESCRIPTION
**Pull request by Pisapou**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Featured_Artists/Featured_Artist_playlists/fr.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+tags:
+  - FA
+  - FA playlist
+  - Artistes Associés
+  - playlist d'Artistes Associés
+---
+
+# Playlists d'Artistes Associés
+
+Tous les mois, de nouvelles **playlists d'Artistes Associés** sont publiés sur [osu!(lazer)](https://github.com/ppy/osu/releases) mettant en valeurs les créations d' [Artistes Associés](/wiki/Featured_Artists) dans tous les [modes de jeu](/wiki/Game_mode). Les playlists ne sont pas soumises à un format strict, ainsi, les paramètres, les sélections de beatmaps, et les critères d'attribution des prix varient chaque mois.
```
Comment (Pisapou) [inline wiki/Featured_Artists/Featured_Artist_playlists/fr.md:11]:
```suggestion
Tous les mois, de nouvelles **playlists d'Artistes Associés** sont publiés sur [osu!(lazer)](https://github.com/ppy/osu/releases) mettant en valeurs les créations d'[Artistes Associés](/wiki/Featured_Artists) dans tous les [modes de jeu](/wiki/Game_mode). Les playlists ne sont pas soumises à un format strict, ainsi, les paramètres, les sélections de beatmaps, et les critères d'attribution des prix varient chaque mois.

```

File: wiki/Featured_Artists/Featured_Artist_playlists/fr.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+!["Lobby de playlist d'Artistes Associés"](img/playlist.jpg "Lobby de playlist d'Artistes Associés sur osu!(lazer)")
+
+Accédez aux playlists via le menu **play** sur osu!(lazer) et en filtrant par **Artistes Associés** dans le coin supérieur droit.
+
+!["Liste de playlist d'Artistes Associés"](img/playlist-listing.jpg)
+
+## Prix
+
+Les meilleurs joueurs de chaque playlist de Featured Artist sont récompensés par des badges de profil spécifiques au mode. Les critères exacts pour les meilleurs joueurs varient selon la liste de lecture.
```
Comment (Pisapou) [inline wiki/Featured_Artists/Featured_Artist_playlists/fr.md:21]:
```suggestion
Les meilleurs joueurs de chaque playlist d'Artiste Associé sont récompensés par des badges de profil spécifiques au mode. Les critères exacts pour les meilleurs joueurs varient selon la liste de lecture.

```

File: wiki/Featured_Artists/Featured_Artist_playlists/fr.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+tags:
+  - FA
+  - FA playlist
+  - Artistes Associés
+  - playlist d'Artistes Associés
+---
+
+# Playlists d'Artistes Associés
+
+Tous les mois, de nouvelles **playlists d'Artistes Associés** sont publiés sur [osu!(lazer)](https://github.com/ppy/osu/releases) mettant en valeurs les créations d'[Artistes Associés](/wiki/Featured_Artists) dans tous les [modes de jeu](/wiki/Game_mode). Les playlists ne sont pas soumises à un format strict, ainsi, les paramètres, les sélections de beatmaps, et les critères d'attribution des prix varient chaque mois.
```
Comment (Y0hananas) [inline wiki/Featured_Artists/Featured_Artist_playlists/fr.md:11]:
```suggestion
Tous les mois, de nouvelles **playlists d'Artistes Associés** sont publiées sur [osu!(lazer)](https://github.com/ppy/osu/releases) mettant en valeur les créations d'[Artistes Associés](/wiki/Featured_Artists) dans tous les [modes de jeu](/wiki/Game_mode). Les playlists ne sont pas soumises à un format strict, ainsi, les paramètres, les sélections de beatmaps et les critères d'attribution des prix varient chaque mois.

```

File: wiki/Featured_Artists/Featured_Artist_playlists/fr.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+
+![badge de profile d'Artistes Associés'](https://assets.ppy.sh/profile-badges/fa-playlists/fapl-osu!.png) ![badge de profile d'Artistes Associés'](https://assets.ppy.sh/profile-badges/fa-playlists/fapl-osu!taiko.png) ![badge de profile d'Artistes Associés'](https://assets.ppy.sh/profile-badges/fa-playlists/fapl-osu!catch.png) ![badge de profile d'Artistes Associés'](https://assets.ppy.sh/profile-badges/fa-playlists/fapl-osu!mania.png)
+
+## Anciennes playlists
+
+### Juin 2022
+
+- 8 beatmaps
+- [Hidden (HD)](/wiki/Game_modifier/Hidden) mod requis
```
Comment (Y0hananas) [inline wiki/Featured_Artists/Featured_Artist_playlists/fr.md:32]:
```suggestion
- [Hidden (HD)](/wiki/Game_modifier/Hidden) mode requis

```

File: wiki/Featured_Artists/Featured_Artist_playlists/fr.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+- [No Fail (NF)](/wiki/Game_modifier/No_Fail) et [Mirror (MR)](/wiki/Game_modifier/Mirror) mods libres
```
Comment (Y0hananas) [inline wiki/Featured_Artists/Featured_Artist_playlists/fr.md:33]:
```suggestion
- [No Fail (NF)](/wiki/Game_modifier/No_Fail) et [Mirror (MR)](/wiki/Game_modifier/Mirror) modes libres

```

File: wiki/Featured_Artists/Featured_Artist_playlists/fr.md [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+- Des prix ont été remis aux 3 meilleurs joueurs de chaque playlist.
+
+### Juillet 2022
+
+- 5 beatmaps
+- 15 tentatives autorisées pour chaque beatmap de la playlist
+- [Hidden (HD)](/wiki/Game_modifier/Hidden), [Hard Rock (HR)](/wiki/Game_modifier/Hard_Rock), [No Fail (NF)](/wiki/Game_modifier/No_Fail) et [Mirror (MR)](/wiki/Game_modifier/Mirror) mods libres
```
Comment (Y0hananas) [inline wiki/Featured_Artists/Featured_Artist_playlists/fr.md:40]:
```suggestion
- [Hidden (HD)](/wiki/Game_modifier/Hidden), [Hard Rock (HR)](/wiki/Game_modifier/Hard_Rock), [No Fail (NF)](/wiki/Game_modifier/No_Fail) et [Mirror (MR)](/wiki/Game_modifier/Mirror) modes libres

```

File: wiki/Featured_Artists/Featured_Artist_playlists/fr.md [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+
+!["Lobby de playlist d'Artistes Associés"](img/playlist.jpg "Lobby de playlist d'Artistes Associés sur osu!(lazer)")
+
+Accédez aux playlists via le menu **play** sur osu!(lazer) et en filtrant par **Artistes Associés** dans le coin supérieur droit.
+
+!["Liste de playlist d'Artistes Associés"](img/playlist-listing.jpg)
+
+## Prix
+
+Les meilleurs joueurs de chaque playlist d'Artiste Associé sont récompensés par des badges de profil spécifiques au mode. Les critères exacts pour les meilleurs joueurs varient selon la liste de lecture.
```
Comment (Biokax) [inline wiki/Featured_Artists/Featured_Artist_playlists/fr.md:23]:
```suggestion
Les meilleurs joueurs de chaque playlist d'Artiste Associé sont récompensés par des badges de profil spécifiques au mode. Les critères exacts pour les meilleurs joueurs varient selon la playlist.

```

File: wiki/Featured_Artists/Featured_Artist_playlists/fr.md [hunk 8]
```diff
@@ -0,0 +1,45 @@
...
+
+
+![badge de profile d'Artistes Associés'](https://assets.ppy.sh/profile-badges/fa-playlists/fapl-osu!.png) ![badge de profile d'Artistes Associés'](https://assets.ppy.sh/profile-badges/fa-playlists/fapl-osu!taiko.png) ![badge de profile d'Artistes Associés'](https://assets.ppy.sh/profile-badges/fa-playlists/fapl-osu!catch.png) ![badge de profile d'Artistes Associés'](https://assets.ppy.sh/profile-badges/fa-playlists/fapl-osu!mania.png)
+
+## Anciennes playlists
+
+### Juin 2022
+
+- 8 beatmaps
+- [Hidden (HD)](/wiki/Game_modifier/Hidden) mode requis
```
Comment (Biokax) [inline wiki/Featured_Artists/Featured_Artist_playlists/fr.md:33]:
```suggestion
- [Hidden (HD)](/wiki/Game_modifier/Hidden) mod requis

```

File: wiki/Featured_Artists/Featured_Artist_playlists/fr.md [hunk 9]
```diff
@@ -0,0 +1,45 @@
...
+
+- [No Fail (NF)](/wiki/Game_modifier/No_Fail) et [Mirror (MR)](/wiki/Game_modifier/Mirror) modes libres
+
+- Des prix ont été remis aux 3 meilleurs joueurs de chaque playlist.
+
+### Juillet 2022
+
+- 5 beatmaps
+- 15 tentatives autorisées pour chaque beatmap de la playlist
+- [Hidden (HD)](/wiki/Game_modifier/Hidden), [Hard Rock (HR)](/wiki/Game_modifier/Hard_Rock), [No Fail (NF)](/wiki/Game_modifier/No_Fail) et [Mirror (MR)](/wiki/Game_modifier/Mirror) modes libres
```
Comment (Biokax) [inline wiki/Featured_Artists/Featured_Artist_playlists/fr.md:43]:
```suggestion
- [Hidden (HD)](/wiki/Game_modifier/Hidden), [Hard Rock (HR)](/wiki/Game_modifier/Hard_Rock), [No Fail (NF)](/wiki/Game_modifier/No_Fail) et [Mirror (MR)](/wiki/Game_modifier/Mirror) options mods libres

```

File: wiki/Featured_Artists/Featured_Artist_playlists/fr.md [hunk 10]
```diff
@@ -0,0 +1,45 @@
...
+![badge de profile d'Artistes Associés'](https://assets.ppy.sh/profile-badges/fa-playlists/fapl-osu!.png) ![badge de profile d'Artistes Associés'](https://assets.ppy.sh/profile-badges/fa-playlists/fapl-osu!taiko.png) ![badge de profile d'Artistes Associés'](https://assets.ppy.sh/profile-badges/fa-playlists/fapl-osu!catch.png) ![badge de profile d'Artistes Associés'](https://assets.ppy.sh/profile-badges/fa-playlists/fapl-osu!mania.png)
+
+## Anciennes playlists
+
+### Juin 2022
+
+- 8 beatmaps
+- [Hidden (HD)](/wiki/Game_modifier/Hidden) mode requis
+
+- [No Fail (NF)](/wiki/Game_modifier/No_Fail) et [Mirror (MR)](/wiki/Game_modifier/Mirror) modes libres
```
Comment (Biokax) [inline wiki/Featured_Artists/Featured_Artist_playlists/fr.md:35]:
```suggestion
- [No Fail (NF)](/wiki/Game_modifier/No_Fail) et [Mirror (MR)](/wiki/Game_modifier/Mirror) options modes libres

```

**Review by Biokax — CHANGES_REQUESTED** [review]

File: wiki/Featured_Artists/Featured_Artist_playlists/fr.md (hunk 10 above)
Comment (Biokax) [reply wiki/Featured_Artists/Featured_Artist_playlists/fr.md:35]: j'ai un doute sur celui là car freemod n'a pas l'air traduis sur les autres pages

**Review by DoryanR — CHANGES_REQUESTED** [review]
In this PR I have the impression that there is a [problem with the formatting of the lines](https://github.com/ppy/osu-wiki/runs/7263466364?check_suite_focus=true) (changing them from (`\r\n`) to (`\n`) in your editor).

For the other errors I have made the necessary corrections normally.

**Comment by DoryanR** [comment]
Personally I was putting `Featured Artists` everywhere, because it seemed right to leave it like that, so I think we can leave it like that (since on the site, it was left like that too)

brave_Sml93MGArT

As for this article, I'll take care of making the change, I remember translating it at the time.

File: wiki/Featured_Artists/Featured_Artist_playlists/fr.md [hunk 11]
```diff
@@ -0,0 +1,42 @@
...
+
+![Badge de profile d'un Featured Artist](https://assets.ppy.sh/profile-badges/fa-playlists/fapl-osu!.png) ![Badge de profile d'un Featured Artist](https://assets.ppy.sh/profile-badges/fa-playlists/fapl-osu!taiko.png) ![Badge de profile d'un Featured Artist](https://assets.ppy.sh/profile-badges/fa-playlists/fapl-osu!catch.png) ![Badge de profile d'un Featured Artist](https://assets.ppy.sh/profile-badges/fa-playlists/fapl-osu!mania.png)
+
+## Anciennes playlists
+
+### Juin 2022
+
+- 8 beatmaps
+- [Hidden (HD)](/wiki/Game_modifier/Hidden) mod requis
+
```
Comment (DoryanR) [inline wiki/Featured_Artists/Featured_Artist_playlists/fr.md:31]:
```suggestion
```

File: wiki/Featured_Artists/Featured_Artist_playlists/fr.md [hunk 12]
```diff
@@ -0,0 +1,42 @@
...
+- [No Fail (NF)](/wiki/Game_modifier/No_Fail) et [Mirror (MR)](/wiki/Game_modifier/Mirror) options mods libres
+
```
Comment (DoryanR) [inline wiki/Featured_Artists/Featured_Artist_playlists/fr.md:33]:
```suggestion
```

File: wiki/Featured_Artists/Featured_Artist_playlists/fr.md [hunk 13]
```diff
@@ -0,0 +1,42 @@
...
+- Des prix ont été remis aux 3 meilleurs joueurs de chaque playlist.
+
+### Juillet 2022
+
+- 5 beatmaps
+- 15 tentatives autorisées pour chaque beatmap de la playlist
+- [Hidden (HD)](/wiki/Game_modifier/Hidden), [Hard Rock (HR)](/wiki/Game_modifier/Hard_Rock), [No Fail (NF)](/wiki/Game_modifier/No_Fail) et [Mirror (MR)](/wiki/Game_modifier/Mirror) options mods libres
+
```
Comment (DoryanR) [inline wiki/Featured_Artists/Featured_Artist_playlists/fr.md:41]:
```suggestion
```

**Review by DoryanR — CHANGES_REQUESTED** [review]
To fix the rest of the CI errors :

File: wiki/Featured_Artists/Featured_Artist_playlists/fr.md [hunk 14]
```diff
@@ -0,0 +1,39 @@
...
+Les meilleurs joueurs de chaque playlist des Featured Artist sont récompensés par des badges de profil spécifiques au mode. Les critères exacts pour les meilleurs joueurs varient selon la playlist.
+
+![Badge de profile d'un Featured Artist](https://assets.ppy.sh/profile-badges/fa-playlists/fapl-osu!.png) ![Badge de profile d'un Featured Artist](https://assets.ppy.sh/profile-badges/fa-playlists/fapl-osu!taiko.png) ![Badge de profile d'un Featured Artist](https://assets.ppy.sh/profile-badges/fa-playlists/fapl-osu!catch.png) ![Badge de profile d'un Featured Artist](https://assets.ppy.sh/profile-badges/fa-playlists/fapl-osu!mania.png)
+
+## Anciennes playlists
+
+### Juin 2022
+
+- 8 beatmaps
+- [Hidden (HD)](/wiki/Game_modifier/Hidden) mod requis
```
Comment (DoryanR) [inline wiki/Featured_Artists/Featured_Artist_playlists/fr.md:30]:
```suggestion
- Mod [Hidden (HD)](/wiki/Game_modifier/Hidden) requis
```

File: wiki/Featured_Artists/Featured_Artist_playlists/fr.md [hunk 15]
```diff
@@ -0,0 +1,39 @@
...
+- [No Fail (NF)](/wiki/Game_modifier/No_Fail) et [Mirror (MR)](/wiki/Game_modifier/Mirror) options mods libres
+- Des prix ont été remis aux 3 meilleurs joueurs de chaque playlist.
+
+### Juillet 2022
+
+- 5 beatmaps
+- 15 tentatives autorisées pour chaque beatmap de la playlist
+- [Hidden (HD)](/wiki/Game_modifier/Hidden), [Hard Rock (HR)](/wiki/Game_modifier/Hard_Rock), [No Fail (NF)](/wiki/Game_modifier/No_Fail) et [Mirror (MR)](/wiki/Game_modifier/Mirror) options mods libres
```
Comment (DoryanR) [inline wiki/Featured_Artists/Featured_Artist_playlists/fr.md:38]:
```suggestion
- Options des mods [Hidden (HD)](/wiki/Game_modifier/Hidden), [Hard Rock (HR)](/wiki/Game_modifier/Hard_Rock), [No Fail (NF)](/wiki/Game_modifier/No_Fail) et [Mirror (MR)](/wiki/Game_modifier/Mirror) libres
```

**Review by DoryanR — DISMISSED** [review]

**Review by Biokax — DISMISSED** [review]

**Review by cl8n — APPROVED** [review]

**Comment by cl8n** [comment]
when applying suggestions, try to name the commit something more descriptive of the changes made

**Pull request merged**